File: android-java/SayfalarArasiGecis/app/src/main/java/com/example/sayfalararasigecis/MainActivity.java
package com.example.sayfalararasigecis;

import androidx.appcompat.app.ActionBarDrawerToggle;
import androidx.appcompat.app.AppCompatActivity;
import androidx.navigation.Navigation;
import androidx.navigation.fragment.NavHostFragment;
import androidx.navigation.ui.NavigationUI;

import android.os.Bundle;

import com.example.sayfalararasigecis.databinding.ActivityMainBinding;
import com.google.android.material.bottomnavigation.BottomNavigationView;
import com.google.android.material.navigation.NavigationView;

public class MainActivity extends AppCompatActivity {
    private ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        NavHostFragment navHostFragment =
                (NavHostFragment) getSupportFragmentManager().findFragmentById(R.id.nav_host_fragment);
        NavigationUI.setupWithNavController(binding.navigatipnView,navHostFragment.getNavController());
        binding.toolbar.setTitle("Başlık");

        ActionBarDrawerToggle toggle =
                new ActionBarDrawerToggle(this,binding.drawer,binding.toolbar,0,0);
        binding.drawer.addDrawerListener(toggle);
        toggle.syncState();


    }
}

File: android-kotlin/IntentApp/app/src/main/java/com/example/intentapp/MainActivity.kt
package com.example.intentapp

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AlertDialog
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat
import com.example.intentapp.databinding.ActivityMainBinding
import org.jetbrains.annotations.ApiStatus.Internal
import java.io.FileInputStream

class MainActivity : AppCompatActivity() {
    private lateinit var binding : ActivityMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        var view = binding.root
        println("hello")
        setContentView(view)
        enableEdgeToEdge()
        binding.nameText.setText("")

    }
    override fun onResume() {
        super.onResume()
        binding.nameText.setText("")
        println("onResume")
    }

    override fun onStart() {
        super.onStart()
        println("onStart")
        binding.nameText.setText("")

    }

    override fun onPause() {
        super.onPause()
        println("onPause")
    }
    override fun onStop() {
        super.onStop()
        println("onStop")
    }
    override fun onDestroy() {
        super.onDestroy()
        println("onDestroy")
    }
    override fun onRestart() {
        super.onRestart()
        println("onRestart")
    }

    fun onNextPageClick (view: View){
        var alert = AlertDialog.Builder(this)
        alert.setTitle("GO TO NEXT PAGE")
        alert.setMessage("Are you sure that you want to go to the next page?")

        alert.setPositiveButton("Yes"){dialog, which ->
            val intent = Intent(this, SecondActivity::class.java)


            intent.putExtra("name",binding.nameText.text.toString())
            startActivity(intent)
        }
        alert.setNegativeButton("No"){dialog, which ->
            dialog.cancel()
        }
        alert.show()




    }

}